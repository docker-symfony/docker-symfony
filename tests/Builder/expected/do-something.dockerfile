FROM image

RUN do something
