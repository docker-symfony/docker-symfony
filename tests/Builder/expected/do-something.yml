version: '3'
services:
    service:
        image: app-image
